platform/appimage/koreader.appdata.xml

```diff
@@ -1,28 +1,40 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component>
-  <id>org.koreader.KOReader</id>
-  <launchable type="desktop-id">org.koreader.KOReader.desktop</launchable>
+  <id>rocks.koreader.KOReader</id>
```

Comment:
I assume this might cause some pain to people who use AppImage installation managers?

Reply:
I'm certainly not opposed to changing it, just trying to figure out the impact. :-)

Reply:
@pazos Okay with you?

Reply:
I'm fine as long as the android id isn't changed :)